package.json

```diff
@@ -44,7 +46,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged; ./node_modules/.bin/eslint . --ext .js"
+      "pre-commit": "lint-staged; ./node_modules/.bin/eslint src/**/*.js --ext .js"
```

Comment:
Just a question -
Why was this edited, was `node_modules` also being checked ?

Reply:
It's to explicitly specify what files to check for lint errors